packages/makecode-core/src/commands.ts

```diff
@@ -914,5 +916,76 @@ export async function shareCommand(opts: ProjectOptions) {
     else {
         error("Unable to share project");
     }
+}
+
+export async function getSimHTML(opts: ProjectOptions) {
+    applyGlobalOptions(opts);
+
+    const project = await resolveProject(opts);
+    const cache = await project.getCacheAsync();
```

Comment:
Is it possible that either of these fail? Just wondering if these fail what would happen to the variable assignments below.